tests/phpunit/item/ItemTest.php

```diff
@@ -61,6 +65,19 @@ public function testItemBlockRegistered() : void{
 		}
 	}
 
+	/**
+	 * Tests that when all enchantments are removed from an item, the "ench" tag is removed as well
+	 */
+	public function testEnchantmentRemoval() : void{
+		$item = ItemFactory::get(Item::DIAMOND_SWORD);
+		$item->addEnchantment(new EnchantmentInstance(Enchantment::getEnchantment(Enchantment::SHARPNESS)));
+		$item->addEnchantment(new EnchantmentInstance(Enchantment::getEnchantment(Enchantment::UNBREAKING)));
+		$item->removeEnchantment(Enchantment::SHARPNESS);
+		self::assertTrue($item->getNamedTag()->hasTag(Item::TAG_ENCH, ListTag::class));
+		$item->removeEnchantment(Enchantment::UNBREAKING);
+		self::assertFalse($item->getNamedTag()->hasTag(Item::TAG_ENCH, ListTag::class));
```

Comment:
This test can be simplified. It only needs to add and remove 1 enchantment.

Reply:
My intention was to test both that the ench is not removed when there are still other enchantments and that it is removed when there are no more enchantments.

Reply:
You don't need to test that. Stick to testing the thing you're trying to fix.